src/form/wrm.jl

```diff
@@ -129,6 +131,21 @@ function constraint_ohms_yt_to{T <: AbstractWRMForm}(pm::GenericPowerModel{T}, f
     return Set([c1, c2])
 end
 
+"""
+Creates Ohms constraints for DC Lines (yt post fix indicates that Y and T values are in rectangular form)
+
+```
+p_fr + p_to == loss0 + loss1 * p_fr
+```
+"""
+function constraint_ohms_yt_dc{T <: AbstractWRMForm}(pm::GenericPowerModel{T}, f_bus, t_bus, f_idx, t_idx, br_status, loss0, loss1)
```

Comment:
Given that this constraint is identical across multiple formulations, you can follow the pattern for constraints like thermal limits. That is move the constraints to `core/constraint.jl` and give it a signature like, `constraint_ohms_yt_dc{T}(pm::GenericPowerModel{T}`, which will be used by any formulation, unless specialized further.